zanata-war/src/main/java/org/zanata/rest/service/ResourceUtils.java

```diff
@@ -1479,4 +1489,11 @@ public static void validateExtensions(Set<String> requestedExt) {
         }
     }
 
+    public static class PluralParseException extends RuntimeException {
```

Comment:
It is, it just lives inside the ResourceUtils namespace.

I could make it a top-level class, but I don't think it's needed.